src/components/ProductFilterModal.vue

```diff
@@ -14,7 +14,7 @@
   </ion-header>
 
   <ion-content>
-    <ion-searchbar :placeholder="$t(`Search ${label}`)" v-model="queryString" @keyup.enter="queryString = $event.target.value; search($event)"/>
+    <ion-searchbar :placeholder="$t(`Search ${label}`)" v-model="queryString" @keyup.enter="isScrollable = true, queryString = $event.target.value, search($event)"/>
```

Comment:
Making `isScrollable = true` on enter will make infiniteScroll active even when we will be having less number of tags than the viewSize and thus an extra API call will be made.